packages/graphback/src/migrations/schema/LocalSchemaProvider.ts

```diff
@@ -0,0 +1,33 @@
+import { ISchemaProvider } from './ISchemaProvider';
+import { buildSchemaText, removeFiles } from '../../utils';
+import { writeFileSync } from 'fs';
+import { join } from 'path';
+
+/**
+ * Provides old and new schema from a local context
+ *
+ * @export
+ * @class LocalSchemaProvider
+ * @extends {ISchemaProvider}
+ */
+export class LocalSchemaProvider implements ISchemaProvider {
+  private oldSchemaDir: string;
+  private newSchemaDir: string;
+  constructor(oldSchemaDir: string, newSchemaDir: string) {
+    this.oldSchemaDir = oldSchemaDir;
+    this.newSchemaDir = newSchemaDir;
+  }
+
+  public getNewSchemaText(): string {
+    return buildSchemaText(this.newSchemaDir);
+  }
+
+  public getOldSchemaText(): string {
+    return buildSchemaText(this.oldSchemaDir);
+  }
+
+  public async updateOldSchema(newSchema: string) {
+    await removeFiles(join(this.oldSchemaDir, '*.graphql'));
```

Comment:
UUU.. kinda risky business. 
Why we need to do removing part? 
Are we ok to leave dangling files and let people manage their repository?

I'm assuming that all `old` ;) schemas are pushed to repo